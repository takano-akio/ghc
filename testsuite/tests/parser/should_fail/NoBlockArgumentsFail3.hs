{-# LANGUAGE LambdaCase #-}
module NoBlockArgumentsFail3 where

forM a b = return ()

foo :: IO ()
foo = forM [1 .. 10] \case
  Just 3 -> print x
