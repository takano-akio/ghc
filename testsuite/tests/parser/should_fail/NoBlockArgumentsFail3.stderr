
NoBlockArgumentsFail3.hs:7:22: error:
    Unexpected lambda-case expression in function application:
        \case Just 3 -> print x
    You could write it with parentheses
    Or perhaps you meant to enable BlockArguments?
